include/itkPyTorchImage.hxx

```diff
@@ -57,11 +54,13 @@ void
 PyTorchImage< TPixel, VImageDimension >
 ::AllocateGPU()
 {
+  // Much of the same work is done in Initialize.  Where should it be done?!!!
```

Comment:
It could be done in a protected/private method, which both `Initialize` and `Allocate` call.